databricks/koalas/series.py

```diff
@@ -1445,7 +1448,7 @@ def to_clipboard(self, excel=True, sep=None, **kwargs):
 
     to_clipboard.__doc__ = DataFrame.to_clipboard.__doc__
 
-    def to_dict(self, into=dict):
+    def to_dict(self, into=dict) -> Dict:
```

Comment:
Oops @xinrong-databricks, sorry I think what you did with `collections.abc.Mapping` is more correct. Looks like `info` might not necessarily be a dict.

Reply:
That's totally fine. 😄  Thank you!